eZ/Publish/API/Repository/SearchService.php

```diff
@@ -18,6 +18,85 @@
 interface SearchService
 {
     /**
+     * Capability flag for scoring feature for use with {@see ::supports()}.
+     *
+     * Scoring, a search feature telling you how well one search hit scores compared to other items in the search result.
+     * When this is supported you can expect search engine to populate SearchHit->score and SearchResult->maxScore
+     * properties as well as sort by this if no sort clauses are specified.
+     */
+    const CAPABILITY_SCORING = 1;
+
+    /**
+     * Capability flag for facets feature for use with {@see ::supports()}.
+     *
+     * Faceted search: https://en.wikipedia.org/wiki/Faceted_search
+     *
+     * Note: Even if search engine tells you this is supported be aware:
+     * - It might not support all facets, by design it will only return facets for the facet builders engine supports.
+     * - Some of the faceting features are still work in process in API and won't be furhter matured before in v2.x
+     *   releases
+     *
+     * @since 6.7.4
+     */
+    const CAPABILITY_FACETS = 2;
+
+    /**
+     * Capability flag for custom fields feature for use with {@see ::supports()}.
+     *
+     * Custom fields is the capability for search engines to 1. allow you to extend the search index via plugins to
+     * generate custom fields, like a different representation (format, ..) of an existing field or similar. And 2.
+     * allow you on some search criteria to specify this custom field to rather query on that instead of the field
+     * generated by the system.
+     */
+    const CAPABILITY_CUSTOM_FIELDS = 4;
+
+    /**
+     * Capability flag for spellcheck feature for use with {@see ::supports()}.
+     *
+     * Spell check within search capabilities refers to abaility to suggest better wordings in fulltext search string.
```

Comment:
`     * Spell check within search capabilities refers to ability to suggest better wordings in fulltext search string.`
(typo in `ability`)